workalendar/america/barbados.py

```diff
@@ -32,14 +37,31 @@ def get_kadooment_day(self, year):
         return (Barbados.get_nth_weekday_in_month(year, 8, MON),
                 "Kadooment Day")
 
+    def get_emancipation_day(self, year):
+        emancipation_day = date(year, 8, 1)
+        if emancipation_day.weekday() == SUN:
```

Comment:
would it be possible to add a few more tests that cover the various "`if`s" of this method, please?
It would validate the fact that it covers all known cases about this new day.

Reply:
I've pushed a new version with tests where Emancipation was a Saturday and when the first of August was a Monday 2020 and 2016 respectively. 
I've noticed a missing pubic holiday in 2016 https://labour.gov.bb/wp-content/pdf//Library/Other-Docs/List-of-Public-Holidays-for-the-Year-2016.pdf so added that one as well. 
So we have cases with emancipation being in a week day (but not monday), then saturday, then sunday and Monday.

Let me know if you think I missed something.

Reply:
I think we're sorted!